CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 2.0.0 (2022-09-02)
+
+### Features
+* support sorting and filtering to iot-table. ([commit hash TBA]())
```

Comment:
hmm, feel like we should say something besides commit hash TBA

Reply:
yeah I think your right. maybe we should always have version bumps (with changelog) separated from the code implementation?

Reply:
no - versions can only be published once. code merged in is not published until a version bump is made

Reply:
Got it. Then we should have code changes and version bumps(change logs) separated.